build fails on AArch64, Fedora 33

[jw@cn06 build]$ make VERBOSE=1 -j$(nproc)
make  all-recursive
make[1]: Entering directory '/data/jw/ovis/build'
Making all in lib
make[2]: Entering directory '/data/jw/ovis/build/lib'
Making all in src
make[3]: Entering directory '/data/jw/ovis/build/lib/src'
Making all in third
make[4]: Entering directory '/data/jw/ovis/build/lib/src/third'
make[4]: Nothing to be done for 'all'.
make[4]: Leaving directory '/data/jw/ovis/build/lib/src/third'
Making all in ovis_util
make[4]: Entering directory '/data/jw/ovis/build/lib/src/ovis_util'
  CC       util.lo
../../../../lib/src/ovis_util/util.c: In function 'ovis_strerror':
../../../../lib/src/ovis_util/util.c:1000:19: error: 'sys_nerr' undeclared (first use in this function)
 1000 |  if (e >=0 && e < sys_nerr)
      |                   ^~~~~~~~
../../../../lib/src/ovis_util/util.c:1000:19: note: each undeclared identifier is reported only once for each function it appears in
../../../../lib/src/ovis_util/util.c:1001:10: error: 'sys_errlist' undeclared (first use in this function)
 1001 |   return sys_errlist[e];
      |          ^~~~~~~~~~~
make[4]: *** [Makefile:672: util.lo] Error 1
make[4]: Leaving directory '/data/jw/ovis/build/lib/src/ovis_util'
make[3]: *** [Makefile:576: all-recursive] Error 1
make[3]: Leaving directory '/data/jw/ovis/build/lib/src'
make[2]: *** [Makefile:505: all-recursive] Error 1
make[2]: Leaving directory '/data/jw/ovis/build/lib'
make[1]: *** [Makefile:624: all-recursive] Error 1
make[1]: Leaving directory '/data/jw/ovis/build'
make: *** [Makefile:532: all] Error 2
[jw@cn06 build]$

(base) [jw@cn06 build]$ make VERBOSE=1 -j1
make  all-recursive
make[1]: Entering directory '/data/jw/ovis/build'
Making all in lib
make[2]: Entering directory '/data/jw/ovis/build/lib'
Making all in src
make[3]: Entering directory '/data/jw/ovis/build/lib/src'
Making all in third
make[4]: Entering directory '/data/jw/ovis/build/lib/src/third'
make[4]: Nothing to be done for 'all'.
make[4]: Leaving directory '/data/jw/ovis/build/lib/src/third'
Making all in ovis_util
make[4]: Entering directory '/data/jw/ovis/build/lib/src/ovis_util'
make[4]: Nothing to be done for 'all'.
make[4]: Leaving directory '/data/jw/ovis/build/lib/src/ovis_util'
Making all in coll
make[4]: Entering directory '/data/jw/ovis/build/lib/src/coll'
make[4]: Nothing to be done for 'all'.
make[4]: Leaving directory '/data/jw/ovis/build/lib/src/coll'
Making all in mmalloc
make[4]: Entering directory '/data/jw/ovis/build/lib/src/mmalloc'
make[4]: Nothing to be done for 'all'.
make[4]: Leaving directory '/data/jw/ovis/build/lib/src/mmalloc'
Making all in ovis_auth
make[4]: Entering directory '/data/jw/ovis/build/lib/src/ovis_auth'
make[4]: Nothing to be done for 'all'.
make[4]: Leaving directory '/data/jw/ovis/build/lib/src/ovis_auth'
Making all in ovis_event
make[4]: Entering directory '/data/jw/ovis/build/lib/src/ovis_event'
make[4]: Nothing to be done for 'all'.
make[4]: Leaving directory '/data/jw/ovis/build/lib/src/ovis_event'
Making all in ovis_ev
make[4]: Entering directory '/data/jw/ovis/build/lib/src/ovis_ev'
make[5]: Entering directory '/data/jw/ovis/build/lib/src/ovis_ev'
make[5]: Nothing to be done for 'all-am'.
make[5]: Leaving directory '/data/jw/ovis/build/lib/src/ovis_ev'
make[4]: Leaving directory '/data/jw/ovis/build/lib/src/ovis_ev'
Making all in zap
make[4]: Entering directory '/data/jw/ovis/build/lib/src/zap'
Making all in .
make[5]: Entering directory '/data/jw/ovis/build/lib/src/zap'
make[5]: Nothing to be done for 'all-am'.
make[5]: Leaving directory '/data/jw/ovis/build/lib/src/zap'
Making all in sock
make[5]: Entering directory '/data/jw/ovis/build/lib/src/zap/sock'
make[5]: Nothing to be done for 'all'.
make[5]: Leaving directory '/data/jw/ovis/build/lib/src/zap/sock'
make[4]: Leaving directory '/data/jw/ovis/build/lib/src/zap'
Making all in ovis_ctrl
make[4]: Entering directory '/data/jw/ovis/build/lib/src/ovis_ctrl'
make[4]: Nothing to be done for 'all'.
make[4]: Leaving directory '/data/jw/ovis/build/lib/src/ovis_ctrl'
Making all in ovis_json
make[4]: Entering directory '/data/jw/ovis/build/lib/src/ovis_json'
make  all-recursive
make[5]: Entering directory '/data/jw/ovis/build/lib/src/ovis_json'
make[6]: Entering directory '/data/jw/ovis/build/lib/src/ovis_json'
make[6]: Nothing to be done for 'all-am'.
make[6]: Leaving directory '/data/jw/ovis/build/lib/src/ovis_json'
make[5]: Leaving directory '/data/jw/ovis/build/lib/src/ovis_json'
make[4]: Leaving directory '/data/jw/ovis/build/lib/src/ovis_json'
make[4]: Entering directory '/data/jw/ovis/build/lib/src'
make[4]: Nothing to be done for 'all-am'.
make[4]: Leaving directory '/data/jw/ovis/build/lib/src'
make[3]: Leaving directory '/data/jw/ovis/build/lib/src'
make[3]: Entering directory '/data/jw/ovis/build/lib'
make[3]: Nothing to be done for 'all-am'.
make[3]: Leaving directory '/data/jw/ovis/build/lib'
make[2]: Leaving directory '/data/jw/ovis/build/lib'
Making all in ldms
make[2]: Entering directory '/data/jw/ovis/build/ldms'
Making all in src
make[3]: Entering directory '/data/jw/ovis/build/ldms/src'
Making all in third-plugins
make[4]: Entering directory '/data/jw/ovis/build/ldms/src/third-plugins'
make[4]: Nothing to be done for 'all'.
make[4]: Leaving directory '/data/jw/ovis/build/ldms/src/third-plugins'
Making all in core
make[4]: Entering directory '/data/jw/ovis/build/ldms/src/core'
make[4]: Nothing to be done for 'all'.
make[4]: Leaving directory '/data/jw/ovis/build/ldms/src/core'
Making all in ldmsd
make[4]: Entering directory '/data/jw/ovis/build/ldms/src/ldmsd'
Making all in .
make[5]: Entering directory '/data/jw/ovis/build/ldms/src/ldmsd'
  CCLD     ldmsd
/usr/bin/ld: ldmsd-ldmsd_auth.o:/data/jw/ovis/build/ldms/src/ldmsd/../../../../ldms/src/ldmsd/ldmsd_auth.c:113: multiple definition of `cfgobj_trees'; ldmsd-ldmsd_cfgobj.o:/data/jw/ovis/build/ldms/src/ldmsd/../../../../ldms/src/ldmsd/ldmsd_cfgobj.c:95: first defined here
collect2: error: ld returned 1 exit status
make[5]: *** [Makefile:781: ldmsd] Error 1
make[5]: Leaving directory '/data/jw/ovis/build/ldms/src/ldmsd'
make[4]: *** [Makefile:1044: all-recursive] Error 1
make[4]: Leaving directory '/data/jw/ovis/build/ldms/src/ldmsd'
make[3]: *** [Makefile:600: all-recursive] Error 1
make[3]: Leaving directory '/data/jw/ovis/build/ldms/src'
make[2]: *** [Makefile:577: all-recursive] Error 1
make[2]: Leaving directory '/data/jw/ovis/build/ldms'
make[1]: *** [Makefile:650: all-recursive] Error 1
make[1]: Leaving directory '/data/jw/ovis/build'
make: *** [Makefile:537: all] Error 2
(base) [jw@cn06 build]$

--- a/ldms/src/ldmsd/ldmsd_auth.c
+++ b/ldms/src/ldmsd/ldmsd_auth.c
@@ -110,7 +110,7 @@ ldmsd_auth_new_with_auth(const char *name, const char *plugin,
 }
 
 ldmsd_cfgobj_t __cfgobj_find(const char *name, ldmsd_cfgobj_type_t type);
-struct rbt **cfgobj_trees;
+extern struct rbt *cfgobj_trees[];
 
 int ldmsd_auth_del(const char *name, ldmsd_sec_ctxt_t ctxt)
 {

